spec/lib/solidus_stripe/configuration_spec.rb

```diff
@@ -11,5 +11,11 @@
     it "defaults webhook signature tolerance to stripe's default" do
       expect(described_class.new.webhook_signature_tolerance).to eq(Stripe::Webhook::DEFAULT_TOLERANCE)
     end
+
+    it "defaults refund reason name to the one in seeds" do
+      expect(
+        described_class.new.default_refund_reason_name
+      ).to eq(SolidusStripe::Seeds::DEFAULT_STRIPE_REFUND_REASON_NAME)
+    end
```

Comment:
Not sure to follow who updates/creates a new refund reason when the preference is updated. Can you help me navigate this?

Reply:
If another name is used, the developer must ensure a corresponding record exists in the database. Our seeds would provide a default, but they can manually change it if needed. Does it make sense?